config/initializers/constants.rb

```diff
@@ -33,3 +33,4 @@
   SETTINGS_EDITOR_APP => 'can_manage_settings'
 }
 PACK_UNPACK_ALLOWED_ACTIONS = %w[pack unpack].freeze
+ALLOWED_MENTIONAblE_ROLES = ['Reviewer', 'Supervisor', 'Order administrator', 'Order fulfilment'].freeze
```

Comment:
```suggestion
MENTIONABLE_ROLES = ['Reviewer', 'Supervisor', 'Order administrator', 'Order fulfilment'].freeze
```
Need to update the name slightly (capitalization)

Also, I wonder if this might be better as a class constant as it puts it in context for the specific role names. I can go either way on this. Your call

```
class Roles
  MENTIONABLE_ROLES = 
```
and accessible as Roles::MENTIONABLE_ROLES